### ISSUE TITLE
Ordered List is not working properly in all android devices

### ISSUE THREAD

**Issue by ThejazR** [wxik/react-native-rich-editor#200]
For example in realme and samsung devices its showing as

1 Apple
1 Orange
1 Mango

Expected
1. Apple
2.  orange
3. mango

Alwasys 1 is coming the subsequent numbers like 2 , 3, .... are not coming.

**Comment by swalahamani** [comment]
I'm also getting the same issue. 

Seems like the editor creates a separate set of `<ul>---</ul>` while tapping either **_return-key_** or **_selecting any suggestion on keyboard_**

**Comment by swalahamani** [comment]
Seems like related to #120

**Comment by ThejazR** [comment]
@swalahamani You can fix this buy disabling the autocorrect or text prediction option from the mobile keyboard.

**Comment by LOMFM** [comment]
@ThejazR Without disabling those options, Can I fix this error?

**Comment by apprisemobile-lyu** [comment]
I am also experiencing this on multiple Android devices. When I add a new number it works correctly until I try to focus the next list item in order to add text.

1.
2.
3.

becomes

1.
2.

1. text here

**Comment by swalahamani** [comment]
> @swalahamani You can fix this by disabling the autocorrect or text prediction option from the mobile keyboard.

Yeah, that is correct. But it seems like the only way to do that is via the settings app and couldn't find any programmatical way.

**Comment by shijinwu** [comment]
in  editor.js     
function formatParagraph(async)     if  (async == true)   return; 
 Can  solved
I don't know if there will be any other problems

**Comment by stulip** [comment]
@shijinwu Can you provide`initialContentHTML` for me to test

**Comment by BadhanGanesh-HappyFox** [comment]
@stulip This does not need initial content. Here are the steps to reproduce though:

1. With the editor empty, type **"Hello".**
2. Press **Enter 2 times**.
3. Tap on the **number list icon**.
4. Type **"One"** and then press enter.
5. Type **"Two"**.

--- 

**Expected Output**

Hello

1. One
2. Two

**Actual Output:**

Hello

1.One

1.Two

**Comment by sujathasperi2022** [comment]
So whats the fix?

**Comment by Kirysha** [comment]
I added my idea to the pull request. You can watch this #314 .
